yarn.lock

```diff
@@ -2593,51 +2575,6 @@ preserve@^0.2.0:
   version "0.2.0"
   resolved "https://registry.yarnpkg.com/preserve/-/preserve-0.2.0.tgz#815ed1f6ebc65926f865b310c0713bcb3315ce4b"
 
-prettier@azz/prettier#f6d0b5fe:
-  version "1.9.2"
-  resolved "https://codeload.github.com/azz/prettier/tar.gz/f6d0b5feceadbc460d05e5ee32ae50b2a071fa55"
-  dependencies:
-    "@babel/code-frame" "7.0.0-beta.35"
-    babylon "7.0.0-beta.34"
-    camelcase "4.1.0"
-    chalk "2.1.0"
-    cjk-regex "1.0.2"
-    cosmiconfig "3.1.0"
-    dashify "0.2.2"
-    diff "3.2.0"
-    editorconfig "0.14.2"
-    editorconfig-to-prettier "0.0.6"
-    emoji-regex "6.5.1"
-    escape-string-regexp "1.0.5"
-    esutils "2.0.2"
-    flow-parser "0.59.0"
-    get-stream "3.0.0"
-    globby "6.1.0"
-    graphql "0.10.5"
-    ignore "3.3.7"
-    jest-docblock "21.3.0-beta.11"
-    jest-validate "21.1.0"
-    leven "2.1.0"
-    mem "1.1.0"
-    minimatch "3.0.4"
-    minimist "1.2.0"
-    parse5 "3.0.3"
-    path-root "0.1.1"
-    postcss-less "1.1.3"
-    postcss-media-query-parser "0.2.3"
-    postcss-scss "1.0.2"
-    postcss-selector-parser "2.2.3"
-    postcss-values-parser "1.3.1"
-    remark-frontmatter "1.1.0"
-    remark-parse "4.0.0"
-    resolve "^1.5.0"
-    semver "5.4.1"
-    string-width "2.1.1"
-    typescript "2.6.2"
-    typescript-eslint-parser "11.0.0"
-    unicode-regex "1.0.1"
-    unified "6.1.6"
-
```

Comment:
@azz not sure why running `yarn` after bumping php-parser caused this change - do you know more?

Reply:
Maybe we forgot to checkin the file last time Prettier was bumped.